coreos-base/oem-packet/files/grub.cfg

```diff
@@ -1,7 +1,6 @@
 # Flatcar GRUB settings
 
 set oem_id="packet"
```

Comment:
(slight offtopic)
@tormath1 ignition now accepts `oem_id=metal` but there are several other spots in init/bootengine repos that still check for `packet`, so this still needs to stay, correct?